.travis.yml

```diff
@@ -43,4 +43,4 @@ install:
     - conda install -n root --yes --quiet jinja2 anaconda-client
 
 script:
-    - conda-build-all ./ --inspect-channels scitools/label/${LABEL_NAME} ${UPLOAD_CHANNELS} --matrix-condition "numpy >=1.8,,<=1.10" "python >=2.7,<3|>=3.4,<3.5|>=3.5,<3.6"
+    - conda-build-all ./ --inspect-channels scitools/label/${LABEL_NAME} ${UPLOAD_CHANNELS} --matrix-condition "numpy >=1.8,,<=1.11" "python >=2.7,<3|>=3.4,<3.5|>=3.5,<3.6"
```

Comment:
This is also causing an issue with building cartopy on OS X so i'm +1, although I'm still investigating. @marqh explained to me that you now need to follow the project on circle ci (just once) to trigger the linux builds for your PRs.